frontend/kesaseteli/employer/src/__tests__/utils/components/render-page.tsx

```diff
@@ -32,6 +34,7 @@ const renderPage = (
             </Layout>
           </ThemeProvider>
         </AuthProvider>
+        <HiddenLoadingIndicator />
```

Comment:
Helper indicator for the tests which will tell if some backend request is till pending